scripts/statistics.js

```diff
@@ -127,6 +127,16 @@ const getStats = (folder, allBrowsers) => {
    */
   const browsers = allBrowsers
     ? Object.keys(bcd.browsers)
+    : folder === 'webextensions'
+    ? [
+        'chrome',
+        'edge',
+        'firefox',
+        'firefox_android',
```

Comment:
Why list Firefox Android and Opera here but not in the list below? The below is used by default, as long as `--all` isn't passed, and it would make sense for this here to be a subset of the default.